### DISCUSSION TITLE
Strong Defaults for Heading Background Colors [IDEA/REQUEST]

### DISCUSSION THREAD

**Discussion by eleanorkonik** [jdanielmourao/obsidian-sanctum#89] [Ideas]
**Is your feature request related to a problem? Please describe.**

The Style Settings defaults for heading colors are black/white but you have a much better sense of color cohesiveness than I do so I trust your judgment more than mine. 

**Describe the solution you'd like**

Let Style Settings come up with defaults for the Font/Background colors 

**Describe alternatives you've considered**

Continuing to use my snippet 😅 

**Additional context**

This is not a high priority ask.

**Comment by jdanielmourao** [comment]
I have been considering making my theme, including the Style Settings options, a bit more opinionated, in order to ensure some cohesiveness in terms of color and graphical language, but I'll have to think about which ones I'm comfortable with restricting, and which ones would be best to just leave it to the user to customise at will.

**Comment by eleanorkonik** [comment]
I've been making an effort to keep the theme as cohesive as possible / stick to your color scheme, so that's fine with me :D Broadly speaking I'm in favor of "strong defaults" or whatever the programming term is.

**Comment by jdanielmourao** [comment]
I've converted this to a discussion. I think it'd be nice to survey people's interest in this, even if it's just to motivate me to add it to the theme.🙃
